Assets/Scripts/Office/VendingMachine/VendingMachineItem.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "VendingMachineItem", menuName = "ScriptableObjects/VendingMachineItem", order = 1)]
+public class VendingMachineItem : ScriptableObject
+{
+    public string itemName;
+    public int price;
+    public int stock;
+    public byte keyCode;
+
+    public Sprite sprite;
+
+    public void PurchaseItem()
+    {
+        PlayerData.playerMoney -= price;
```

Comment:
go through a method instead of editing the money manually. we are gonna have to be REALLY careful about this